app/src/main/java/com/example/datastoreapp/UserInfo.kt

```diff
@@ -0,0 +1 @@
+package com.example.datastoreapp

data class UserInfo(
    val name: String,
    val age: Int
)

```

Comment:
Is this file okay? I never seen before de ^M

Reply:
I tried to configure the Indentation but in the code I don't see these issues. Maybe it's a indentation difference between the OS (I have windows)
![image](https://github.com/SofiaZarzabal/DataStoreApp/assets/84337427/b5f51b2e-8816-4b15-ab20-d7d7d96fbc1c)